frappe/__init__.py

```diff
@@ -1749,15 +1754,13 @@ def parse_json(val):
 	return parse_json(val)
 
 def mock(type, size=1, locale='en'):
```

Comment:
we don't even need this!

Reply:
Will remove that. I have used this to show how lazy_import is used.